bin/sl-pm.js

```diff
@@ -34,7 +34,8 @@ function printHelp($0, prn) {
 
 var argv = process.argv;
 var $0 = process.env.CMD ? process.env.CMD : path.basename(argv[1]);
-var parser = new Parser([
+var parser = new Parser(
+  [
```

Comment:
ok, I'm ok with it. One think I've noted with lint... it encourages me to break up complex constructions like that into
```
var args = [
  ...
];
var parser = new Parser(args.join(''), argv);
```
because I hate its opinion about formatting of complex function calls.
anyhow, I'm ok with your change, and the lint rules that led to it.